psalm.xml

```diff
@@ -0,0 +1,20 @@
+<?xml version="1.0"?>
+<psalm
+    errorLevel="2"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xmlns="https://getpsalm.org/schema/config"
+    xsi:schemaLocation="https://getpsalm.org/schema/config vendor/vimeo/psalm/config.xsd"
+    errorBaseline="psalm-baseline.xml"
+>
+    <projectFiles>
+        <directory name="src" />
+        <ignoreFiles>
+            <directory name="vendor" />
+        </ignoreFiles>
+    </projectFiles>
+	<issueHandlers>
+		<PropertyNotSetInConstructor errorLevel="suppress"/>
+		<UndefinedConstant errorLevel="suppress"/>
+		<MissingClosureParamType errorLevel="suppress"/>
```

Comment:
I added these 3 general suppress rules to block unnnecessary warnings like `WWW_ROOT is not defined` or `not all params have a type definition`